clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/DefaultLaunchConfigurationBuilder.groovy

```diff
@@ -144,12 +142,14 @@ class DefaultLaunchConfigurationBuilder implements LaunchConfigurationBuilder {
 
       String existingAppGroup = names.keySet().find { it.contains(application) }
       if (!existingAppGroup) {
-        String applicationSecurityGroup = securityGroupService.getSecurityGroupForApplication(application, subnetType)
-        if (!applicationSecurityGroup) {
-          applicationSecurityGroup = securityGroupService.createSecurityGroup(application, subnetType)
-        }
+        OperationPoller.retryWithBackoff({ ->
+          String applicationSecurityGroup = securityGroupService.getSecurityGroupForApplication(application, subnetType)
+          if (!applicationSecurityGroup) {
+            applicationSecurityGroup = securityGroupService.createSecurityGroup(application, subnetType)
+          }
 
-        securityGroupIds << applicationSecurityGroup
+          securityGroupIds << applicationSecurityGroup
+        }, 500, 3);
```

Comment:
I think 500ms is good, but if not, it'll retry 1000ms later, then once more 2000ms later before failing. And, yeah, it is going against AWS directly.